distribution/docs/src/main/resources/content/_configuring/exporting-configurations.adoc

```diff
@@ -1,63 +0,0 @@
-:title: Exporting Existing Configurations
```

Comment:
❓ Interesting. I thought we had already removed that section. Is this something we need to backport to 2.11.x?

Reply:
I have no idea how it got re-introduced as I had removed them. They do not show up in the documentation as far as I could tell but I might be mistaken.